### PR TITLE
feat: 🎸 support hex input in internal `asBaseAsset` method

### PR DESCRIPTION
**Pull request by polymath-eric**
### Description

support hex input in a missed internal method

### Checklist

- [ ] Updated the Readme.md (if required) ?

### CODE REVIEW

File: src/api/client/Assets.ts [hunk 1]
```diff
@@ -345,6 +346,7 @@ export class Assets {
     const { context } = this;
 
     const assetIdValue = await this.getIdFromAssetIdAndTicker(assetId, ticker);
+    console.log({ assetIdValue });
```
Comment (F-OBrien) [inline src/api/client/Assets.ts:349]: console.log should be removed

**Review by VictorVicente — APPROVED** [review]

**Pull request merged**